src/dropbox.js

```diff
@@ -66,6 +66,24 @@ export default class Dropbox {
   }
 
   request(path, args, auth, host, style) {
+    let multiAuth;
+    if (auth.split(',').length > 1) {
+      // if there are multiple auth types, creates set with different auth types
+      const authTypes = auth.replace(' ', '').split(',');
+      multiAuth = new Set(authTypes);
+    }
+    // checks which auth types are in the multiauth and assigns one based on priority
+    // to create header in switch case
+    if (multiAuth) {
```

Comment:
Can this blob be lumped into the other if statement? 
So it would just be 
```
if (auth.split...) {
 const authTypes = 
  if authTypes.has...
  
}

Reply:
yea sounds good to me